### PR TITLE
Issues/summary drop node

### PR DESCRIPTION
**Pull request by gijskant**
- Fix dropping nodes in summary statistics
- Report number of exported rows.

### CODE REVIEW

File: grails-app/services/com/recomdata/transmart/asynchronous/job/AsyncJobService.groovy [hunk 1]
```diff
@@ -209,6 +210,10 @@ class AsyncJobService {
 	  } else {
 	  	  result.put('statusIndexExists', false)
 	  }
+      def rowsWritten = jobResultsService[jobName]["RowsWritten"]
+      if (rowsWritten) {
+          result.put("rowsWritten", rowsWritten)
+      }
```
Comment (cataphract) [inline grails-app/services/com/recomdata/transmart/asynchronous/job/AsyncJobService.groovy:217]: whhitespace is off

File: grails-app/services/com/recomdata/transmart/asynchronous/job/AsyncJobService.groovy [hunk 2]
```diff
@@ -50,6 +50,7 @@ class AsyncJobService {
 			m["startDate"] = jobResult.lastRunOn
 			m["viewerURL"] = jobResult.viewerURL
 			m["altViewerURL"] = jobResult.altViewerURL
+            m["rowsWritten"] = new JSONObject(jobResultsService[jobResult.jobName]?.get("RowsWritten"))
```
Comment (JanKanis) [inline grails-app/services/com/recomdata/transmart/asynchronous/job/AsyncJobService.groovy]: whitespace

File: web-app/js/datasetExplorer/exportData/exportJobsTab.js [hunk 3]
```diff
@@ -66,6 +66,16 @@ function exportjobsstoreLoaded()
...
+                  {name:'rowsWritten', header: "Written", width: 120, sortable: false, dataIndex: 'rowsWritten',
+                      renderer: function (value, metaData, record, rowIndex, colIndex, store) {
+                          var keys = Object.keys(value).sort();
+                          return jQuery.map(keys, function (key) {
+                              var val = value[key];
+                              var ext = key.split('.').pop();
+                              return ext + ' (' + val + ' rows)';
+                          }).join(', ');
+                      }
+                  },
```
Comment (JanKanis) [inline web-app/js/datasetExplorer/exportData/exportJobsTab.js:78]: whitespace

File: web-app/js/myJobs.js [hunk 4]
```diff
@@ -171,6 +172,7 @@ function showJobStatusWindow(result, messages)	{
 		        	  }
 		          },
 		          {
+		              id: 'background-button',
```
Comment (JanKanis) [inline web-app/js/myJobs.js:175]: more whitespace

**Comment by JanKanis** [comment]
The functionality of HighDimDatatypeController already exists in HighdimController. But using that will create a dependency from transmartApp to transmart-rest-api. IMO that is not a real problem since in the long term we would like the UI to be based on the rest api anyway.

**Comment by JanKanis** [comment]
But Gustavo thought: I don't think it's a good idea if no other reason than the rest api needs to be accessed via oauth

So it's not quite going to work.

**Comment by gijskant** [comment]
@cataphract @JanKanis concerning the whitespace comments: these files already have inconsistent whitespacing. Actually, my additions are relatively consistent compared to the rest. 
Alternatively, I could do a whitespace cleanup completely, but since we would like to merge stuff later, that might not be a good idea.

**Pull request merged**